storage/innobase/xtrabackup/doc/source/conf.py

```diff
@@ -52,7 +52,7 @@
 # The short X.Y version.
 version = '8.0'
 # The full version, including alpha/beta/rc tags.
-release = '8.0.4'
+release = '8.0.5-1'
```

Comment:
As far as I know we haven't change release numbering rules. The release number should be 8.0.5 and -1 is the part of the package build number.

Reply:
Ready for review